### PR TITLE
Mobile UI: make SsrfSwitch resizable

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [X] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Back in 8ab8a67f the checkbox where made resizable.
This applies the same functionality to the selector switch and makes the
two objects match in vertical size.

### CODE REVIEW

**Comment by jbygdell** [comment]
Derek add label: ui-mobile

**Comment by jbygdell** [comment]
Derek add label: complexity-low

File: mobile-widgets/qml/SsrfSwitch.qml [hunk 1]
```diff
@@ -18,8 +18,8 @@ Switch {
 		Rectangle {
 			x: root.checked ? parent.width - width : 0
 			y: parent.height / 2 - height / 2
-			width: Kirigami.Units.largeSpacing * 1.1
-			height: Kirigami.Units.largeSpacing * 1.1
+			width: 20 * PrefDisplay.mobile_scale
+			height: 20 * PrefDisplay.mobile_scale
```
Comment (dirkhh) [inline mobile-widgets/qml/SsrfSwitch.qml:22]: not disagreeing with the change, but what I find odd is that line 9 to line 21 used to have about 2x difference, vs. it now is a 1.8 difference.
I assume there's a good reason for that and it now looks better? :-)

**Comment by jbygdell** [comment]
Given that the current dize of the switch is a pinhead in comparison to the checkbox. The logical choice was to make them equal in height, inner and outer. While keeping as close as possible to the original ratio of the switch.

**Comment by dirkhh** [comment]
That makes sense. Thanks

**Pull request merged**